backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminProcessor.java

```diff
@@ -143,10 +149,30 @@ public synchronized void addConcept(@NonNull Dataset dataset, @NonNull Concept<?
 			throw new WebApplicationException("Can't replace already existing concept " + concept.getId(), Status.CONFLICT);
 		}
 
+		// Add selects automatically
+		addAutomaticSelect(concept, EventDurationSumSelect::new);
+
 		datasetRegistry.get(dataset.getId()).getStorage().updateConcept(concept);
 		datasetRegistry.get(dataset.getId()).sendToAll(new UpdateConcept(concept));
 	}
 
+	private static void addAutomaticSelect(@NotNull Concept<?> concept, Supplier<UniversalSelect> selectCreator) {
+		if (concept instanceof TreeConcept) {
```

Comment:
ne das sollte schon in der conf gemacht werden

Reply:
meinst du neva_conf oder ConqueryConfig?

Reply:
Ist jetzt erstmal nicht schneller um zu setzten

Reply:
neva_conf